source/integrate-with-integration-environment/authenticate-your-user.html.md.erb

```diff
@@ -208,7 +208,7 @@ Location: https://YOUR_REDIRECT_URI?error=invalid_request
 |-----------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
 | `unauthorized_client`       | Your service is not authorised to request an authorisation code.<br>Make sure you have [registered your service to use GOV.UK One Login][integrate.register-your-service].                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       |
 | `invalid_request`           | The request has one or more of the following issues: <ul><li>missing a required parameter</li><li>includes an invalid parameter value</li><li>includes a parameter more than once</li><li>not in the correct format</li></ul><br>You can [check which parameters GOV.UK One Login supports when you make an authorisation request][integrate.make-authorization-request].                                                                                                                                                                                                                                                                                                                                                                                                                                                              |
-| `invalid_request - Request vtr not valid`| If you're logging in without second factor authentication (requesting medium confidence ('P2')), you must request a protection level of Cl.Cm or you will receive this error.
+| `invalid_request - Request vtr not valid`| You've logged in without two-factor authentication and requested a medium confidence level (`P2`). To fix, you must request a protection level of `Cl.Cm`.
```

Comment:
This doesn't sound quite right. This is `You've requested single factor authentication and a medium confidence level` or similar

Reply:
Past tense on "logged in" makes it incorrect. The P and C are requested at the same time, and the issue here is that the two values are incompatible

Reply:
cheers @ethanmills - good points. I was trying to make this error consistent with the other error codes appearing in the docs (i.e. stating what's happened and how to fix it). Would the following be an accurate way of describing this error: 

_You've requested single factor authentication and a medium confidence level (`P2`). Make sure you are requesting two-factor authentication and `P2`._

Happy to jump on a quick call if easier than threading.